src/index.html

```diff
@@ -5,8 +5,380 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="nav">
+      <a href="#" class="nav__logo">
```

Comment:
It's header (not navigation)

```suggestion
    <header class="header">
      <a href="#" class="header__logo">
```